mne_qt_browser/_pg_figure.py

```diff
@@ -69,6 +69,28 @@ def _get_std_icon(icon_name):
         getattr(QStyle, icon_name))
 
 
+def _get_color(color_spec):
+    """Wraps mkColor to accept all possible matplotlib color-specifiers."""
+    try:
+        # Convert matplotlib color-names if possible
+        color_spec = _to_rgb(color_spec, alpha=True)
+    except ValueError:
+        pass
+
+    # Convert tuples of floats from 0-1 to 0-255 for pyqtgraph
+    if (isinstance(color_spec, tuple) and
+            all([i <= 1 for i in color_spec])):
+        color_spec = tuple([int(i * 255) for i in color_spec])
+
+    try:
+        color = mkColor(color_spec)
+    except ValueError:
+        raise ValueError(f'"{color_spec}" is not a valid matplotlib '
+                         f'color-specifier!')
```

Comment:
```suggestion
                         f'color-specifier!') from None
```
because it's cleaner:
```
>>> try:
...     raise ValueError
... except Exception:
...     raise RuntimeError
... 
Traceback (most recent call last):
  File "<stdin>", line 2, in <module>
ValueError

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "<stdin>", line 4, in <module>
RuntimeError
>>> try:
...     raise ValueError
... except Exception:
...     raise RuntimeError from None
... 
Traceback (most recent call last):
  File "<stdin>", line 4, in <module>
RuntimeError
```